includes/gateways/stripe/includes/admin/admin-actions.php

```diff
@@ -388,3 +456,96 @@ function give_stripe_show_currency_notice() {
 }
 
 add_action( 'admin_notices', 'give_stripe_show_currency_notice' );
+
+/**
+ * Disconnect Stripe Account.
+ *
+ * @since 2.7.0
+ *
+ * @return void
+ */
+function give_stripe_disconnect_connect_stripe_account() {
+	$get_data = give_clean( $_GET );
+
+	if ( is_admin() && isset( $get_data['stripe_disconnected'] ) ) {
+		$account_name = ! empty( $get_data['account_name'] ) ? $get_data['account_name'] : false;
+
+		// Disconnect Stripe Account.
+		give_stripe_disconnect_account( $account_name );
+	}
+}
+
+add_action( 'admin_init', 'give_stripe_disconnect_connect_stripe_account' );
+
+/**
+ * Set default Stripe account.
+ *
+ * @since 2.7.0
+ *
+ * @return void
+ */
+function give_stripe_set_account_default() {
+	$post_data    = give_clean( $_POST );
```

Comment:
@mehul0810 always uses user capability + nonce when integrating with the database. Logic like this can provide easy access to any anonymous.